assets/sass/components/comments.scss

```diff
@@ -0,0 +1,27 @@
+.comments-area {
```

Comment:
@nucro please edit the markup for the comments section also, add appropriate class names and then use them for the scss here following BEM and SMACCS.  You can reference the example that I left in the header.scss file.